### PR TITLE
Modify stringTokenizer to avoid use of magic literals

### LINKED ISSUE #60: Code Review for @mcmc101001
`src/main/java/cashleh/parser/StringTokenizer.java`
* LIne 31, 32, 35: I observe that you have several lines which check for ":optional", consider assigning this to a variable then referencing when needed to reduce usage of magic strings

### PR DESCRIPTION
**Pull request by mcmc101001**

### CODE REVIEW

**Review by Xuan127 — APPROVED** [review]
LGTM

File: src/main/java/cashleh/parser/StringTokenizer.java
Comment (danielpappa) [inline src/main/java/cashleh/parser/StringTokenizer.java:1]: Good refactoring to avoid magic literals

**Review by danielpappa — APPROVED** [review]
LGTM

**Review by WooKaiNing — APPROVED** [review]
LGTM, code is improved with removal of magic strings

**Pull request merged**